src/contract-deployment/deploy.ts

```diff
@@ -16,10 +16,23 @@ export interface DeployResult {
 export const deploy = async (
   config: RollupDeployConfig
 ): Promise<DeployResult> => {
-  const AddressManager: Contract = await getContractFactory(
-    'Lib_AddressManager',
-    config.deploymentSigner
-  ).deploy()
+  let AddressManager: Contract
+
+  if (config.addressManager) {
```

Comment:
Allows us to use an existing address manager instead of doing a fresh deploy every time.